src/einsteinpy/plotting/geodesic/interactive.py

```diff
@@ -173,21 +186,25 @@ def plot(self, geodesic, color="#{:06x}".format(random.randint(0, 0xFFFFFF))):
         )
 
         _title_3D = dict(
-            text="Geodesic Plot", y=0.9, x=0.46, xanchor="center", yanchor="top"
+            text=title, y=0.9, x=0.46, xanchor="center", yanchor="top"
         )
 
         _scene_3D = dict(
             xaxis_title="X (GM/c^2)", yaxis_title="Y (GM/c^2)", zaxis_title="Z (GM/c^2)"
         )
 
-        self.fig.update_layout(title=_title_3D, scene=_scene_3D)
+        if aspect=="manual":
```

Comment:
Just `self.fig.update_layout(title=_title_3D, scene=_scene_3D, scene_aspectmode=aspect, scene_aspectratio=aspect_ratio)` would work, since `scene_aspectratio` only plays a role when mode is set to `manual`. It does not result in errors irrespective of mode's setting.